neard/Cargo.toml

```diff
@@ -28,6 +28,7 @@ futures = "0.3"
 nearcore = { path = "../nearcore" }
 near-primitives = { path = "../core/primitives" }
 near-performance-metrics = { path = "../utils/near-performance-metrics" }
+testlib = { path = "../test-utils/testlib" }
```

Comment:
This feels like something we should not do.

In my mind, `testlib` is something we can use solely for testing, so it generally can depend on random stuff, and needn't be developed as carefully as production code. At the same time, this dependency means that we are going to link this testing code into the actual binary we ship to users.